packages/assets-controllers/src/NftController.ts

```diff
@@ -385,17 +292,25 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
     tokenId: string,
   ): Promise<NftMetadata> {
     // TODO Parameterize this by chainId for non-mainnet token detection
-    // Attempt to fetch the data with the proxy
-    const nftInformation: OpenSeaV2GetNftResponse | undefined =
+    // Attempt to fetch the data with the nft-api
+    const urlParams = new URLSearchParams({
+      chainIds: '1',
+      tokens: `${contractAddress}:${tokenId}`,
+      includeTopBid: 'true',
+      includeAttributes: 'true',
+      includeLastSale: 'true',
+    }).toString();
+    const nftInformation: ReservoirResponse | undefined =
       await fetchWithErrorHandling({
-        url: this.getNftApi({
-          contractAddress,
-          tokenId,
-        }),
+        url: `${this.getNftApi()}?${urlParams}`,
+        options: {
+          headers: {
+            Version: '1',
+          },
+        },
       });
-
     // if we were still unable to fetch the data we return out the default/null of `NftMetadata`
-    if (!nftInformation?.nft) {
+    if (!nftInformation?.tokens[0].token) {
```

Comment:
Maybe more optional chaining in case more fields are missing?

```suggestion
    if (!nftInformation?.tokens?.[0]?.token) {
```